ch02.asciidoc

```diff
@@ -0,0 +1,40 @@
+[[ch02_getting_started]]
+== Getting Started
+
+// TODO intro paragraph - no empty headings!
+
+==== Choosing a Lightning Network Wallet
+
+* full nodes (c-lightning, eclair, lnd) + remote controls
+* phone / desktop wallets (SPV clients)
+* custodial services / wallets?
+// Mastering bitcoin also had a section about custodial web wallets. So it might be fair to include them.
```

Comment:
yes but I kind of didn't like that section about mastering bitcoin (:

Reply:
There really aren't any "web" wallets for Lightning as is today, but there're custodial wallets that are mobile-only (BlueWallet, Wallet of Satoshi, etc).